lib/aur-build

```diff
@@ -106,7 +106,7 @@ while true; do
             repo_add_args+=(-s) ;;
         # chroot options
         -D|--directory)
-            shift; makechrootpkg_args+=(-r "$1") ;;
+            shift; chroot_args+=(-D "$1") ;;
```

Comment:
When we experimented with `aurbuild`: 9e709249b6c23465729056bc1edcf36748263b74 [L83](https://github.com/AladW/aurutils/commit/9e709249b6c23465729056bc1edcf36748263b74?diff=unified#diff-9ae9c47a4e26cbd989e2a9e7b916abe5L83-R87) `-D` was moved from `chroot_args` to `aurbuild_args`. Apparently we reverted it incorrectly.